exercises/pythagorean-triplet/src/test/java/PythagoreanTripletTest.java

```diff
@@ -20,6 +20,156 @@ public void shouldCalculateSum() {
+        for (int i = 0; i < expected.size(); ++i) {
+            assertEquals(expected.get(i), actual.get(i));
+        }
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void tripletsWhoseSumIs90() {
+        List<PythagoreanTriplet> actual
+                = PythagoreanTriplet
+                        .makeTripletsList()
+                        .withFactorsLessThanOrEqualTo(90)
+                        .thatSumTo(90)
+                        .build();
+
+        List<PythagoreanTriplet> expected
+                = Arrays.asList(
+                        new PythagoreanTriplet(9, 40, 41),
+                        new PythagoreanTriplet(15, 36, 39));
+
+        assertEquals(expected.size(), actual.size());
+        for (int i = 0; i < expected.size(); ++i) {
+            assertEquals(expected.get(i), actual.get(i));
+        }
+    }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void tripletsWhoseSumIs840() {
+        List<PythagoreanTriplet> actual
+                = PythagoreanTriplet
+                        .makeTripletsList()
+                        .withFactorsLessThanOrEqualTo(840)
+                        .thatSumTo(840)
+                        .build();
+
+        List<PythagoreanTriplet> expected
+                = Arrays.asList(
+                        new PythagoreanTriplet(40, 399, 401),
+                        new PythagoreanTriplet(56, 390, 394),
+                        new PythagoreanTriplet(105, 360, 375),
+                        new PythagoreanTriplet(120, 350, 370),
+                        new PythagoreanTriplet(140, 336, 364),
+                        new PythagoreanTriplet(168, 315, 357),
+                        new PythagoreanTriplet(210, 280, 350),
+                        new PythagoreanTriplet(240, 252, 348));
+
+        assertEquals(expected.size(), actual.size());
+        for (int i = 0; i < expected.size(); ++i) {
+            assertEquals(expected.get(i), actual.get(i));
+        }
+	}
+
+    @Ignore("Remove to run test")
+    @Test
+    public void tripletsWhoseSumIs30000() {
+        List<PythagoreanTriplet> actual
+                = PythagoreanTriplet
+                        .makeTripletsList()
+                        .withFactorsLessThanOrEqualTo(30000)
+                        .thatSumTo(30000)
+                        .build();
+
+        List<PythagoreanTriplet> expected
+                = Arrays.asList(
+                        new PythagoreanTriplet(1200, 14375, 14425),
+                        new PythagoreanTriplet(1875, 14000, 14125),
+                        new PythagoreanTriplet(5000, 12000, 13000),
+                        new PythagoreanTriplet(6000, 11250, 12750),
+                        new PythagoreanTriplet(7500, 10000, 12500));
+
+        assertEquals(expected.size(), actual.size());
+        for (int i = 0; i < expected.size(); ++i) {
+            assertEquals(expected.get(i), actual.get(i));
+        }
+    }
+
     @Ignore("Remove to run test")
```

Comment:
This test and all the ones below it don't appear to be in the canonical data? In which case they should be removed 🙂